doc/api/http2.md

```diff
@@ -179,15 +179,15 @@ immediately following the `'frameError'` event.
 added: v8.4.0
 -->
 
-The `'goaway'` event is emitted when a `GOAWAY` frame is received. When invoked,
-the handler function will receive three arguments:
-
 * `errorCode` {number} The HTTP/2 error code specified in the `GOAWAY` frame.
 * `lastStreamID` {number} The ID of the last stream the remote peer successfully
   processed (or `0` if no ID is specified).
 * `opaqueData` {Buffer} If additional opaque data was included in the `GOAWAY`
   frame, a `Buffer` instance will be passed containing that data.
 
+The `'goaway'` event is emitted when a `GOAWAY` frame is received. When invoked,
+the handler function will receive three arguments:
```

Comment:
The second sentence is now obsolete and should be removed.

Reply:
I have no idea why I missed this in the first place. Sorry, will do.